events/hfe_events_pirate.txt

```diff
@@ -1998,8 +2001,12 @@ country_event = {
 				factor = 2
 				is_authoritarian = yes
 			}
-			modifier = {
+      modifier = {
```

Comment:
Why this not intended

Reply:
Just as slavers want slaves, Xenophiles will want to RESCUE slaves.

Reply:
I mean tab intended